### ISSUE TITLE
ARIA attributes missing

### ISSUE THREAD

**Issue by patrick-wolf** [vakata/jstree#528]
Hi,

any plans to apply the pull request https://github.com/vakata/jstree/pull/359 which added ARIA attributes to the 3.0 code path? I do see that 3.0 uses the "role" attribute, but is missing the ARIA attributes like "aria-expanded" or "aria-labelledby". This would help a lot to get better accessibility for screen reader users.

Regards
Patrick
